packages/paste-website/src/pages/patterns/status/index.mdx

```diff
@@ -350,187 +446,38 @@ Connectivity statuses are used to show if a user, object, or system is online or
   </CalloutText>
```

Comment:
chore: remove this Callout. It's too specific to Flex, and not really relevant to any other use case.